runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -272,8 +272,17 @@ private NamespaceConfig reconfigure(
         {
             NamespaceConfig oldNamespace = namespaces.get(configURL);
             configurationManager.unregister(oldNamespace);
-            configurationManager.register(newNamespace);
-            namespaces.put(configURL, newNamespace);
+            try
+            {
+                configurationManager.register(newNamespace);
+                namespaces.put(configURL, newNamespace);
+            }
+            catch (Exception ex)
+            {
+                context.onError(ex);
```

Comment:
What is an example of an exception that can be caught here?

Reply:
Basically almost any exception that we can get at startup. What I faced: changed the configfile, so the keystore location is pointing to a non-existing file. In this case, we get FileNotFoundException.